dependencies.gradle

```diff
@@ -0,0 +1,5 @@
+// Add your dependencies here
+
+dependencies {
+
```

Comment:
It had three libraries in libs but no dependencies?

Reply:
They were debugging utilities only